From 8bdce6319bff9d580e7089fcc728bb8aae32aab4 Mon Sep 17 00:00:00 2001
From: zeroskyx <zeroskyx@users.noreply.github.com>
Date: Sun, 12 May 2024 19:46:19 +0200
Subject: [PATCH] hwdb: updated Librem 11 accelerometer (#32772)

(cherry picked from commit fa8f2a948cfb4992ea2f12c692a88dde312b107a)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index aec8a13252..b45db6292b 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -962,9 +962,10 @@ sensor:modalias:acpi:SMO8500*:dmi:*:svnProwise:pnPT301:*
 # Purism
 #########################################
 
-# Librem11
-sensor:modalias:acpi:MXC6655*:dmi:*:svnPurism*:pnlibrem_11*
- ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+# Librem 11 (SeaBIOS + PureBoot)
+sensor:modalias:acpi:MXC6655*:dmi:*:svnPurism:pnLibrem11:*
+sensor:modalias:acpi:MXC6655*:dmi:*:svnPurism:pnlibrem_11:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, -1
 
 #########################################
 # Reeder
